CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 3.0.3
```

Comment:
@kevmoo two tests added, the other methods are for OAuth providers only.

Version updated.